src/vs/gitpod/browser/workbench/workbench.ts

```diff
@@ -964,10 +964,9 @@ async function doStart(): Promise<IDisposable> {
 		urlCallbackProvider: new LocalStorageURLCallbackProvider(),
 		credentialsProvider,
 		productConfiguration: {
-			nameShort: product.nameShort + (info.ideAlias === 'code-latest' ? ' - Insiders' : ''),
-			nameLong: product.nameLong + (info.ideAlias === 'code-latest' ? ' - Insiders' : ''),
-			version: product.version + (info.ideAlias === 'code-latest' ? '-insider' : ''),
-			quality: (info.ideAlias === 'code-latest' ? 'insider' : 'stable'),
+			nameShort: product.nameShort + (product.quality === 'insider' ? ' - Insiders' : ''),
+			nameLong: product.nameLong + (product.quality === 'insider' ? ' - Insiders' : ''),
+			version: product.version,
```

Comment:
Let's do this also in the dockerfile, so I can copy it for openvscode-server release action, and not modify upstream code